public/app/features/alerting/unified/components/TimeToNow.tsx

```diff
@@ -0,0 +1,15 @@
+import { dateTimeFormatTimeAgo, DateTimeInput } from '@grafana/data';
+import React, { FC, useEffect, useState } from 'react';
+
+export interface Props {
+  date: DateTimeInput;
+}
+
+export const TimeToNow: FC<Props> = ({ date }) => {
+  const setRandom = useState(0)[1];
+  useEffect(() => {
+    const interval = setInterval(() => setRandom(Math.random()), 1000);
```

Comment:
Could you explain what this effect is doing?

Reply:
it's setting state every second to force a re-render and update the date readout